src/components/organisms/MailContainer.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="flex">
```

Comment:
We can reuse message container for this. instead of swapping out the containers swap out the ConversationWindow component

Reply:
Benoit and I talked about it and we couldn't immediately find a way to pass in a child component, so we decided to implement separate components instead for the time being.

Reply:
We can do what you are currently doing by commenting out the component but inside of the container